tests/48admin.pl

```diff
@@ -0,0 +1,31 @@
+test "/whois",
+   requires => [ $main::API_CLIENTS[0] ],
+
+   do => sub {
+      my ( $http ) = @_;
```

Comment:
Any reason this isn't

    requires => [ local_user_fixture() ],
    do => sub {
       my ( $user ) = @_;

?